Examples/Example.Logging/Example.Logging.csproj

```diff
@@ -1,12 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="RockLib.Logging" Version="3.0.2" />
-  </ItemGroup>
-
-
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
We are keeping this example here?  Or should we bump to a dotnet 6 example?

Reply:
I'll update the examples to .NET 8 once all of the libraries are published to NuGet